edwin/client/static/js/stores/BugStore.js

```diff
@@ -211,6 +235,20 @@ BugStore.dispatchToken = Dispatcher.register((action) => {
       BugStore.emitChange();
       break;
 
+    case ActionTypes.SET_BLOCKER_BUGS:
+      // Merge newBugs into bugMap
+      for (let bug of action.newBugs) {
+        bugMap = bugMap.update(bug.id, new Immutable.Map(), oldBug => {
+          let newBug = Immutable.fromJS(bug);
+          return oldBug.merge(newBug);
+        });
```

Comment:
I think these 4 lines could be replaced with `bugMap = bugMap.mergeIn([bug.id], Immutable.fromJS(bug))`;

I think.